core/spec/models/spree/order/adjustments_spec.rb

```diff
@@ -26,4 +26,53 @@
       expect(persisted_order.state).to eq('payment')
     end
   end
+
+  context 'when an order has an taxed shipment with tax included_in_price and apply free_shipping_promotion' do
+    let!(:order) { create(:order) }
+    let!(:line_item) { create(:line_item) }
```

Comment:
@factor4 you can create a line item already tied to order, eg.

```
let!(:line_item) { create(:line_item, order: order) }
```

Reply:
Thanks for merging! :) Of course I can change this. If have done it on my local machine and the example still pass. Should I push it in my forks branch? Does this work if you already merged the PR?

Reply:
@factor4 both solutions are OK! Just a reference for future PRs :)

Reply:
Ok! I keep that in mind! :thumbsup: